### ISSUE TITLE
Add anomaly time series

### ISSUE THREAD

**Issue by signeaa** [metno/sea-ice-index-viz#129]
For next development, it would be very useful to add the option of showing anomaly wrt 1981-2010 (or 1919-2020). It should be relatively "straight forward" I think ...(?)

**Comment by huaracheguarache** [comment]
Yes, calculating this should be relatively simple, but the question is how we're going to implement it in terms of an interface. I don't think we'll be able to implement it in the form of a button/dropdown menu to change between the regular view and the anomaly. That's because there is not a full overlap in the legends between the two views. I think we'll need to create a separate tool for this.

**Comment by signeaa** [comment]
I see your point. Could a solution be to add a button on the original interface that switched to another tool page with a similar layout? That is, a button that works as a URL link to a different page. In this way, we could keep it simple with only the daily and monthly entrances. But with the buttons to switch between absolute and anomaly plots. Maybe this does not make sense or is not possible...

**Comment by signeaa** [comment]
Nothing new, but just uploaded a figure with monthly anomaly timeseries (the right subplot) which I find more and more useful. It is difficult to make a similar plot with the absolute values.
![image004](https://github.com/metno/sea-ice-index-viz/assets/38102185/1822510f-c681-462b-8550-cedd3fd9826a)

**Comment by huaracheguarache** [comment]
Anomaly plots have now been added for daily plots in #148. Monthly anomalies are TBD.

**Comment by TomLav** [comment]
In the anomaly plot, the Hover tool:

- change the text from "Index" to "Anomaly" (or "Anom." if too long).
- always have a sign for the anomaly number.
